test/components/DateRangePickerInputController_spec.jsx

```diff
@@ -401,6 +536,74 @@ describe('DateRangePickerInputController', () => {
           });
         });
       });
+
+      describe('is the same day as props.endDate', () => {
+        const endDate = moment(today).add(5, 'days');
+
+        describe('props.minimumNights is 0', () => {
+          it('calls props.onDatesChange with provided start date and props.endDate', () => {
+            const onDatesChangeStub = sinon.stub();
+            const wrapper = shallow(
+              <DateRangePickerInputController
+                onDatesChange={onDatesChangeStub}
+                endDate={endDate}
+                minimumNights={0}
+              />);
+            wrapper.instance().onStartDateChange(validFutureDateString);
+            expect(onDatesChangeStub.callCount).to.equal(1);
+
+            const onDatesChangeArgs = onDatesChangeStub.getCall(0).args[0];
+            const futureDate = moment(validFutureDateString);
+            expect(isSameDay(onDatesChangeArgs.startDate, futureDate)).to.equal(true);
+            expect(onDatesChangeArgs.endDate).to.equal(endDate);
+          });
+        });
+
+        describe('props.minimumNights is greater than 0', () => {
+          it('calls props.onDatesChange with provided start date and null end date', () => {
+            const onDatesChangeStub = sinon.stub();
+            const wrapper = shallow(
+              <DateRangePickerInputController
+                onDatesChange={onDatesChangeStub}
+                endDate={endDate}
+                minimumNights={1}
+              />);
```

Comment:
similarly, since the opening `(` ends a line, the closing `)` should start a line.